packages/babel-plugin-transform-flow-comments/src/index.ts

```diff
@@ -255,7 +255,7 @@ export default declare(api => {
 
       Class(path) {
         const { node } = path;
-        let comments = [];
+        let comments: [string?, ...(string | t.Comment)[]] = [];
```

Comment:
The linter throws error on later usage:
```js
comments[0] += ` ${comment}`;
```
because we also push `t.Comment` to `comments`.

Here we narrow the type so that the first element of `comments` must be a string.